src/appState.ts

```diff
@@ -92,7 +92,7 @@ const APP_STATE_STORAGE_CONF = (<
 >(
   config: { [K in keyof T]: K extends keyof AppState ? T[K] : never },
 ) => config)({
-  appearance: { browser: true, export: false },
+  appearance: { browser: true, export: true },
```

Comment:
not positive about this since export already seems to take appearance into account separately? I left the toggle on there as a way to maybe swap it before you export, I think that's a reasonable feature.

Reply:
`export` here means when you save to `.excalidraw` file (the JSON). It should be named (though there's a jsdoc above). And in this case we don't want to persist a user preference to the JSON file.